Source/WebCore/Modules/modern-media-controls/controls/inline-media-controls.js

```diff
@@ -209,7 +209,7 @@ class InlineMediaControls extends MediaControls
         this.rightContainer.layout();
 
         const widthLeftOfTimeControl = this.leftContainer.children.length > 0 ? this.leftContainer.width : this.leftContainer.leftMargin;
-        const widthRightOfTimeControl = this.rightContainer.children.length > 0 ? this.rightContainer.width : this.rightContainer.rightMargin;
+        const widthRightOfTimeControl = shownRightContainerButtons.length > 0 ? this.rightContainer.width : this.rightContainer.rightMargin;
```

Comment:
i think you'll need to reevaluate `shownRightContainerButtons` cause we `button.dropped = true` inside the `for` above which will (eventually) result in `button.visible` being set to `false`